mmhuman3d/models/registrants/smplify.py

```diff
@@ -783,5 +774,43 @@ def _expand_betas(self, batch_size, betas):
         return betas_video
 
     @staticmethod
-    def _compute_relative_loss_change(pre_v, cur_v):
+    def _compute_relative_change(pre_v, cur_v):
+        """Compute relative loss change. If relative change is small enough, we
+        can apply early stop to accelerate the optimization.
+
+        Args:
+            pre_v: previous value
+            cur_v: current value
+
+        Returns:
+            float: relative change
+        """
         return (pre_v - cur_v) / max([np.abs(pre_v), np.abs(cur_v), 1])
```

Comment:
How's `1` determined here?

Reply:
It is an empirical fomula but I haven't find a reference yet.

When both values are smaller than 1, it degrades to abosolute change. Intuitively, if two values are small and close, dividing the difference by the max value will yield a large value. For example, when `pre_v=0.0000011` and `cur_v=0.000001`,  `pre_v - cur_v = 1e-7` while `(pre_v - cur_v) / pre_v = 0.09`. The desciption has been added to https://github.com/open-mmlab/mmhuman3d/pull/146/commits/954b88e746b6a97de90f34f320ee23a02f073807.

Besides, to avoid the optimization stops immediately when `pre_v < cur_v`, the numerator has been revised to `np.abs(pre_v - cur_v)`.